backend/routers/field_officer.py

```
@@ -438,37 +438,22 @@ def get_visit_statistics(db: Session = Depends(get_db)):
             return Response(content=cached_json, media_type="application/json")
 
         # Optimized: Use a single aggregate query to fetch multiple statistics in one database roundtrip
-        agg_stats = db.query(
+        stats = db.query(
             func.count(func.distinct(FieldOfficerVisit.officer_email)).label('unique_officers'),
-            func.avg(FieldOfficerVisit.distance_from_site).label('avg_distance')
+            func.avg(FieldOfficerVisit.distance_from_site).label('avg_distance'),
+            func.count(FieldOfficerVisit.id).label('total_visits'),
+            func.sum(case((FieldOfficerVisit.verified_at.isnot(None), 1), else_=0)).label('verified_visits'),
+            func.sum(case((FieldOfficerVisit.within_geofence == True, 1), else_=0)).label('within_geofence_count'),
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `FieldOfficerVisit.within_geofence:` for truth checks

Replace with `FieldOfficerVisit.within_geofence`

(E712)